public/emails/generic.liquid

```diff
@@ -0,0 +1,7 @@
+---
+from: "{{ data.from }}"
+layout: modules/core/mailer
+subject: "{{ data.subject }}"
+to: "{{ data.to }}"
+---
+{% include data.partial, data: data %}
```

Comment:
Use `render`

Reply:
we can't use dynamic partial names with render